### ISSUE TITLE
Method column_styles is missing

### ISSUE THREAD

**Issue by afittro** [caxlsx/caxlsx#352]
The method is defined in this repo in lib/axlsx/workbook/worksheet/worksheet.rb:
```
def column_styles(*styles)
  styles.each_with_index do |style, index|
    next if style.nil?
    Axlsx.validate_unsigned_int(style)
    find_or_create_column_info(index).style = style
  end
end
```

My Gemfile includes 'caxlsx' and it is version 4.0.0
![image](https://github.com/caxlsx/caxlsx/assets/541872/6e1da089-9a85-4d46-9d96-1a62b154cba1)

I would run `@sheet.column_styles(*styles.map { |s| @formats[s] })` and my code would complain that column_styles was not a valid method name.

My local copy of worksheet.rb went from column_widths to col_style definitions, completely skipping column_styles.

When I added your code for column_styles from this very repository into my local worksheet.rb in the gem source, things worked flawlessly.

My environment is ruby 3.2.2 running on either Windows or WSL/Ubunutu. The problem occurred in both places.

**Comment by kiskoza** [comment]
Hi. This code was added in since the latest release ([in PR #317](https://github.com/caxlsx/caxlsx/pull/317)), it's going to be included in the next (probably 4.1.0) release. I can't tell you an exact date yet, but looking at the Changelog, we have a few PRs merged already

**Comment by kiskoza** [comment]
The method should be available in caxlsx 4.1.0, released on February 26th
